Farmer/Farmer.gd

```diff
@@ -40,6 +44,22 @@ func _physics_process(delta):
 			if (can_shoot):
 				shoot_at(body.get_global_position())
 
+func update_sprite(direction):
+	if direction.length() == 0:
+		$Sprite/AnimationPlayer.stop()
+	else:
+		if !$Sprite/AnimationPlayer.is_playing():
+			$Sprite/AnimationPlayer.play("farmer_walk_right")
+	if direction.y <= 0:
```

Comment:
Can we check the angle and only face up if it's noticeably up to avoid the thing you pointed out in the PR description?

Reply:
ooo good idea